src/ekore/operator_matrix_elements/unpolarized/time_like/__init__.py

```diff
@@ -1,15 +1,58 @@
-r"""The polarized, time-like |OME|."""
+r"""The unpolarized, time-like |OME|."""
 
 import numba as nb
+import numpy as np
+
+from . import as1
 
 
 @nb.njit(cache=True)
-def A_non_singlet(_matching_order, _n, _sx, _nf, _L):
-    """Compute the non-singlet |OME|."""
-    raise NotImplementedError("Time-like is not yet implemented")
+def A_non_singlet(matching_order, N, L):
+    r"""Compute the non-singlet |OME|.
+
+    Parameters
+    ----------
+    matching_order : tuple(int, int)
+        perturbative matching order
+    N : complex
+        Mellin moment
+    L : float
+        :math:`\ln(\mu_F^2 / m_h^2)`
+
+    Returns
+    -------
+    A_non_singlet : numpy.ndarray
+        non-singlet |OME|
+
+    """
+    if matching_order[0] > 1:
+        raise Exception("Time-like matching conditions are only known upto NLO")
```

Comment:
Maybe it's better to have a warning here: imagine you want to do the best we have, then you do NNLO evolution + NLO matching, but this will prevent to do it...

Reply:
So for this, I have not changed it to either of the two things we discussed that day but rather I have used a method which was used in MAP fragmentation functions, as suggested by Emanuele. Basically now the as1 matching conditions are used for both NLO and NNLO ADs.

Reply:
yes indeed this is the best you can do